cocos2d/CCScheduler.m

```diff
@@ -137,19 +300,20 @@ -(void)setPaused:(BOOL)paused
 @end
 
 
-@interface NSNull(CCSchedulerTarget)<CCSchedulerTarget>
+@interface NSNull(CCSchedulableTarget)<CCSchedulableTarget>
 @end
 
 
-@implementation NSNull(CCSchedulerTarget)
+@implementation NSNull(CCSchedulableTarget)
 -(NSInteger)priority {return NSIntegerMax;}
 @end
 
 
+//MARK: CCTimer
```

Comment:
Actually I'd do it the other way around. The pragma version came first, then the comment version. I think the reason was because pragmas are supposed to be for compilers, not IDEs. You are actually supposed to put ifdefs around pragmas since they are compiler specific and will cause an errors in other compilers. (Not that it's a problem here)